src/locale/oc.js

```diff
@@ -0,0 +1,89 @@
+//! moment.js locale configuration
+//! locale : Occitan, lengadocian dialecte [oc-lnc]
+//! author : Quentí
+
+;(function (global, factory) {
```

Comment:
Please change this file to match the format as in 
https://github.com/moment/moment/blob/develop/src/locale/en-gb.js

Then, the Travis build should complete.

Reply:
Thanks! But I keep doing it wrong apparently =/